### DISCUSSION TITLE
v0.0.1-beta

### DISCUSSION THREAD

**Discussion by KakoozaJerry** [osckampala/First-contribution-practice#16] [Announcements]
## What's Changed
* Adds FAQS for the open source community. by @rodrickcalvin in https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/pull/2
* patrick's changes by @patrickf949 in https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/pull/6
* Add Hassan Bahati Mukisa by @HassanBahati in https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/pull/1
* Benjamin Rukundo added his name by @rukundob451 in https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/pull/8
* [Rodrick] Adds name to Readme by @rodrickcalvin in https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/pull/3
* Refactor by @oscakampala in https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/pull/10
* Simple Refactor by @oscakampala in https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/pull/11
* Formatting Changes by @katungi in https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/pull/13
* Recommendation to use github desktop by @mark-ssentongo in https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/pull/14
* Was able to add my name by @rukundob451 in https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/pull/12

## New Contributors
* @rodrickcalvin made their first contribution in https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/pull/2
* @patrickf949 made their first contribution in https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/pull/6
* @HassanBahati made their first contribution in https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/pull/1
* @rukundob451 made their first contribution in https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/pull/8
* @oscakampala made their first contribution in https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/pull/10
* @katungi made their first contribution in https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/pull/13
* @mark-ssentongo made their first contribution in https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/pull/14

**Full Changelog**: https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/commits/v0.0.1-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/releases/tag/v0.0.1-beta'>v0.0.1-beta</a>.</em>

**Comment by rukundob451** [comment]
Thanks for this @KakoozaJerry.
